metrics/measured_http/http.go

```diff
@@ -62,11 +62,37 @@ func (h *MeasuredHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	begin := h.clk.Now()
 	rwws := &responseWriterWithStatus{w, 0}
 
+	// Use the method string only if it's a recognized HTTP method. This avoids
+	// ballooning timeseries with invalid methods from public input.
+	var method string
+	switch r.Method {
+	case http.MethodGet:
```

Comment:
Could you collapse these into a single case (i.e. `case http.MethodGet, http.MethodHead, ...`), using multiple `fallthrough`s is a bit confusing.